projects/ngx-speculoos/src/lib/route.ts

```diff
@@ -251,6 +256,16 @@ class ActivatedRouteSnapshotStub extends ActivatedRouteSnapshot {
     return convertToParamMap(this.queryParams);
   }
 
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
```

Comment:
I think I already asked in the past, but I don't remember. Why do we need this?

Reply:
Because TypeScript generates an error if we override a property declared in the base class (a simple property, not a getterà), with a getter.